shared/validation/index.js

```diff
@@ -68,66 +70,67 @@ function validateSentence(validator, sentence) {
     validationResult.error = 'Contains multiple sentences';
     return validationResult;
   }
-  
+
   if (!validateWithoutEnglishCharacters(validator, sentence)) {
     validationResult.error = 'Contains english characters';
     return validationResult;
   }
-  
+
+
 
   return validationResult;
 }
 
 function validateCorrectLength(validator, sentence) {
   const result =
-    typeof validator.filterNumbers !== 'function' ?
-      DEFAULT_VALIDATOR.filterLength(sentence) :
-      validator.filterLength(sentence);
+      typeof validator.filterNumbers !== 'function' ?
```

Comment:
please revert these changes (and all below that have to additional intendation), let's keep coding style consistent